### PR TITLE
Added apache-libcloud to support GCE (Google Cloud).

### PR DESCRIPTION
**Pull request by rmarsagoogle**
The gce module is a core ansible module, but it can't be used without apache-libcloud.  This patch adds that library as a python dependency so this will work.

### CODE REVIEW

File: Library/Formula/ansible.rb [hunk 1]
```diff
@@ -204,6 +204,14 @@ class Ansible < Formula
     sha256 "3e95445c1db500a344079a47b171c45ef18f57d188dffdb0e4165c71bea8eb3d"
   end
 
+	#
+	# apache libcloud (for Google GCE cupport)
+	#
```
Comment (mikemcquaid) [inline Library/Formula/ansible.rb]: Indentation here needs fixed.

Comment (rmarsagoogle) [inline Library/Formula/ansible.rb]: This is fixed.  Do I need to submit a new pull request?


Robert Marsa | Senior Staff Software Engineer | rmarsa@google.com <mailto:rmarsa@google.com> |

Comment (bfontaine) [inline Library/Formula/ansible.rb]: @rmarsagoogle No need to, you can update this pull-request by pushing new commits to it.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `ansible: add apache-libcloud to support GCE.`.

**Pull request closed without merging**